python/revng/restapi/globals.py

```diff
@@ -0,0 +1,9 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from pathlib import Path
+from .schema import SchemafulManager
+
+manager: SchemafulManager | None = None
+workdir: Path | None = None
```

Comment:
I think these should be defined in the same file where they are initialized

Reply:
`globals` is gone